### ISSUE TITLE
specify install location using build.sh

### ISSUE THREAD

**Issue by jinz2014** [microsoft/onnxruntime#4428]
Please advise how to specify a custom install location using build.sh. I don't have root permission. 

Thank you

Install the project...
-- Install configuration: "RelWithDebInfo"
-- Installing: /usr/local/include/onnxruntime/core/common
CMake Error at cmake_install.cmake:41 (file):
  file INSTALL cannot make directory
  "/usr/local/include/onnxruntime/core/common": No such file or directory

**Comment by jinz2014** [comment]
Found the answer.

**Comment by jf99** [comment]
> Found the answer.

Which is ...?

**Comment by jinz2014** [comment]
May the onnx runtime developer update the build guide for the question ?

**Comment by askhade** [comment]
You can override the default location by adding an extra cmake argument
--cmake_extra_defines CMAKE_INSTALL_PREFIX=<location>

**Comment by jinz2014** [comment]
Thank you for your answer.

**Comment by Maturin** [comment]
I ran into the same issue, but with a Windows build. While reading the build.py script I also found the 

    --cmake_extra_defines CMAKE_INSTALL_PREFIX=<my path>

option. But building using the build.bat/build.py combination does not execute the install target. Since Visual Studio/msbuild is used for building underneath, one option would be to open the generated solution. But I think this cumbersome to build everything by using scripts and using the Visual Studio for installing the libraries. 

Or did I miss an option for invoking the install target by using the build scripts.
